test/filt_stream.jl

```diff
@@ -12,7 +12,7 @@ function naivefilt(h::Vector, x::Vector, resamplerate::Rational=1//1)
         xZeroStuffed[n*upfactor+1] = x[n+1]
     end
 
-    y = Base.filt(h, one(eltype(x)), xZeroStuffed)
```

Comment:
What was the intention of the `Base` qualification anyway? Has `DSP.filt` ever been a separate function, not just additional methods to `Base.filt`?